docker-compose.yml

```diff
@@ -199,7 +199,7 @@ services:
 
   # OpenMRS 3 Frontend
   frontend:
-    image: openmrs/openmrs-reference-application-3-frontend:${O3_FRONTEND_TAG:-nightly}
+    image: openmrs/openmrs-reference-application-3-frontend:3.0.0-beta.8
```

Comment:
We will need to add this `3.0.0-beta.8` appropriately to the quick start guide and the manual setup instructions as well.

Reply:
@Ruhanga  This is now resolved